scripts/generate_json_api.py

```diff
@@ -365,6 +365,26 @@ async def play(self):
 
         await self.daemon.jsonrpc_comment_abandon(reply['comment_id'])
 
+        # collections
+        collectiontom = await r(
```

Comment:
in Python the standard practice is to separate words in variables with underscores. this should be `collection_tom`, but what is a `tom`?